care/emr/migrations/0030_facilityreporttemplate.py

```
@@ -0,0 +1,39 @@
+    operations = [
+        migrations.CreateModel(
+            name='FacilityReportTemplate',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('external_id', models.UUIDField(db_index=True, default=uuid.uuid4, unique=True)),
+                ('created_date', models.DateTimeField(auto_now_add=True, db_index=True, null=True)),
+                ('modified_date', models.DateTimeField(auto_now=True, db_index=True, null=True)),
+                ('deleted', models.BooleanField(db_index=True, default=False)),
+                ('history', models.JSONField(default=dict)),
+                ('meta', models.JSONField(default=dict)),
+                ('type', models.CharField(max_length=100)),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)